pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.poetry]
+name = "demisto-py"
+version = "3.3.1beta2"
+description = "\"A Python library for the Demisto SDK\""
+authors = ["Demisto"]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [{include = "demisto_client"}]
+homepage = "https://github.com/demisto/demisto-py"
+keywords=["Swagger", "Demisto API"]
+classifiers=[
+        'Intended Audience :: Developers',
+        'Natural Language :: English',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',    
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: Implementation :: CPython'
+    ]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.11"
+certifi = ">=2017.4.17"
+six = ">=1.10"
+python-dateutil = ">=2.5.3"
+setuptools = ">=21.0.0"
```

Comment:
@dorschw do we need to delete the `setuptools`?

Reply:
yes

Reply:
Done